transformato/mutate.py

```diff
@@ -120,13 +124,16 @@ def perform_mutations(
     # Turn off hydrogens
 
```

Comment:
can you add an assert statement that ensures that nr_of_mutation_steps_lj_of_hydrogens is >= 1?